db/schema.rb

```diff
@@ -146,17 +146,17 @@
     t.index ["organization_id"], name: "index_donation_sites_on_organization_id"
   end
 
-  create_table "donations", force: :cascade do |t|
+  create_table "donations", id: :serial, force: :cascade do |t|
     t.string "source"
-    t.bigint "donation_site_id"
+    t.integer "donation_site_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.bigint "storage_location_id"
+    t.integer "storage_location_id"
     t.text "comment"
     t.integer "organization_id"
     t.integer "diaper_drive_participant_id"
     t.datetime "issued_at"
-    t.integer "money_raised"
+    t.float "money_raised"
```

Comment:
The migration says that it wants to `ChangeDonationMoneyRaisedTypeToFloat`, and this is listed as changing to a float, but [the change to the column](https://github.com/rubyforgood/diaper/pull/1353/files#diff-3ace391c0143fad4c9a540f3cb9448caR5) (probably) correctly says to change the datatype to a decimal. It looks like you've already figured out that it should be decimal (with a precision of 2) instead of float, but the migration does not match.